basics/indexing/bloom-filter.md

```diff
@@ -1,22 +1,26 @@
+---
+description: This page describes configuring the bloom filter for Apache Pinot
+---
+
 # Bloom Filter
 
-Bloom filter helps prune segments that do not contain any record matching an **EQUALITY** predicate.
+The bloom filter prunes segments that do not contain any record matching an **EQUALITY** predicate.
 
-It would be useful for a query like the following:
+This is useful for a query like the following:
 
-```
+```sql
 SELECT COUNT(*) 
 FROM baseballStats 
 WHERE playerID = 12345
 ```
 
-There are 3 parameters to configure the Bloom Filter:
+There are 3 parameters to configure the bloom filter:
 
-* `fpp`: False positive probability of the bloom filter (from `0` to `1`, `0.05` by default). The lower the `fpp` , the higher accuracy the bloom filter has, but it will also increase the size of the bloom filter.
-* `maxSizeInBytes`: Maximum size of the bloom filter (unlimited by default). If a certain `fpp` generates a bloom filter larger than this size, we will increase the `fpp` to keep the bloom filter size within this limit.
-* `loadOnHeap`: Whether to load the bloom filter using heap memory or off-heap memory (`false` by default).
+* **`fpp`**: False positive probability of the bloom filter (from `0` to `1`, `0.05` by default). The lower the `fpp` , the higher accuracy the bloom filter has, but it will also increase the size of the bloom filter.
+* **`maxSizeInBytes`**: Maximum size of the bloom filter (unlimited by default). If a `fpp` setting generates a bloom filter larger than this size, using this setting will increase the `fpp` to keep the bloom filter size within this limit.
```

Comment:
2. Set the maximum size of the bloom filter `maxSizeInBytes` (unlimited by default)....